validation_tests/cinema/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+. ./setup.sh
+
+#
+# clone the repository we expect 
+#
+echo "Cloning repository for $CINEMASCI $CINEMASCI_VERSION ..."
+git clone $CINEMASCI_REPOSITORY $CINEMASCI_TEST_DIR
+pushd $CINEMASCI_TEST_DIR
+git checkout $CINEMASCI_COMMIT
+popd
+
+#
+# load the cinemasci module 
+#
+spackLoadUnique py-cinemasci@1.3
```

Comment:
For reasons I am still investigating this fails when run from the top level driver script. The package load needs to be in setup.sh.

Reply:
Will do.

Reply:
Not critical for this first pass, but one thing to consider is if it's possible for clean to delete just the artifact(s) of running the test instead of the whole test directory. So subsequent runs could detect and reuse the test directory instead of doing the git operation every time.

Reply:
I moved the call to spackLoadUnique, but I have the opposite behavior: my test fails when I have the load in the setup.sh script, and works when it is in the run.sh.

I'm unable to test in this configuration, but if it works for you, we can check this in.

Reply:
Are you launching it by doing: `./test-all.sh validation_tests/cinema/`  from the top level testsuite directory? Please send me the error you're seeing. This works for me so I'm merging but I'll try to get it working on your side too.